finesapp/src/main/kotlin/ua/anastasiia/finesapp/dto/response/CarResponse.kt

```diff
@@ -0,0 +1,29 @@
+package ua.anastasiia.finesapp.dto.response
+
+import org.springframework.data.mongodb.core.mapping.Field
+import ua.anastasiia.finesapp.entity.MongoFine
+
+data class CarResponse(
+    @Field("car.plate")
```

Comment:
if it is dto, why do you need spring data annotations here?

Reply:
it was a hack👀 which helped me to get all cars using include() in find() in MongoTemplate. deleted it and used aggregation instead